package.json

```diff
@@ -3,30 +3,35 @@
   "private": true,
   "scripts": {
     "standardjs": "standard",
-    "stylelint": "stylelint **/*.scss",
-    "test": "jest"
+    "stylelint": "stylelint app/assets/stylesheets/**/*.scss",
+    "test": "jest",
+    "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --public-path=assets --target=es6",
+    "build:css": "sass ./app/assets/stylesheets/application.scss:./app/assets/builds/application.css --no-source-map --load-path=node_modules"
   },
   "dependencies": {
-    "@rails/activestorage": "^7.0.4",
     "@rails/ujs": "^7.0.4",
```

Comment:
Is `@rails/ujs` dependency still required?

Reply:
Yeah, it is. 

You can see the [related test failures when I tried to remove it](https://app.circleci.com/pipelines/github/ministryofjustice/laa-apply-for-legal-aid/20382/workflows/19c5c664-e4a9-4cc7-86b9-05653cb87835/jobs/106031) 😢